EdgeMultiplay/Scripts/DataContracts.cs

```diff
@@ -490,6 +501,29 @@ public class Player
         /// </summary>
         [DataMember]
         public int playerAvatar = 0;
+
+        /// <summary>
+        /// helper instance variable for serializing and deserializing playerTags
+        /// </summary>
+   		[DataMember (EmitDefaultValue = false)]
+        internal Hashtable playerTags;
+
+        /// <summary>
+        /// Dictionary<string,string> custom data associated with the player
+        /// </summary>
+        public Dictionary<string, string> playerTagsDict {
+            get 
+            {
+                if(playerTags != null)
```

Comment:
nit space.

Reply:
It's the default spacing on my VS, I am planning to update spacing in a separate PR once I figure out how to change it.